features/machine/machine_healthcheck.feature

```diff
@@ -0,0 +1,30 @@
+Feature: MachineHealthCheck Test Scenarios
+
+  # @author jhou@redhat.com
+  # @case_id OCP-25897
+  @admin
+  @destructive
+  Scenario: Remediation should be applied when the unhealthyCondition 'Ready' is met
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I pick a random machineset to scale
+    # Create MHC
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/cloud/mhc/mhc1.yaml" replacing paths:
+      | n                                                                                  | openshift-machine-api       |
+      | ["metadata"]["name"]                                                               | mhc-<%= machine_set.name %> |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]    | <%= machine_set.cluster %>  |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"] | <%= machine_set.name %>     |
+    Then the step should succeed
+    Given I register clean-up steps:
```

Comment:
@jhou1 perhaps use this existing step instead?  `Given I ensure "mch-#{machine_set.name}" machinehealthcheck is deleted`?

Reply:
@pruan-rht Thanks for the comment, I updated it now.